screens/WalletDetailScreen.tsx

```diff
@@ -10,13 +10,16 @@ import {
   ScrollView,
   StyleSheet,
   TouchableOpacity,
-  View
+  View,
+  Text
```

Comment:
Not using `Text` in the app, please use the app specific `T` for consistent sizing